pkg/storage/export/export/export_test.go

```diff
@@ -1302,10 +1303,13 @@ var _ = Describe("Export controller", func() {
 		}
 		Expect(instancetypeInformer.GetStore().Add(testInstanceType)).To(Succeed())
 
-		res, err := controller.expandVirtualMachine(vm)
+		res, err := controller.generateVMDefinitionFromVm(vm)
```

Comment:
should probably rename the test name from: "Should properly expand instance types of VMs"
to "Should properly generate VM definition with instance type"